dove/tests/cmd_dove_test_fail.rs

```diff
@@ -0,0 +1,121 @@
+#![cfg(test)]
+
+use std::path::{Path, PathBuf};
+use std::fs::{remove_dir_all, read_to_string};
+use fs_extra::file::write_all;
+use dove::cli::execute;
+
+/// $ dove test
+/// project name: demoproject_12
+#[test]
+fn fail() {
+    // Path to dove folder
+    let dove_folder = {
+        let mut folder = Path::new(".").canonicalize().unwrap();
+        if folder.to_str().unwrap().find("dove").is_none() {
+            folder.push("dove");
+        }
+        folder
+    };
+    // Project name and path
+    let project_name = "demoproject_12";
+    let project_folder = {
+        let mut folder = dove_folder.clone();
+        folder.push(project_name);
+        if folder.exists() {
+            remove_dir_all(&folder).expect(&format!(
+                "[ERROR] Couldn't delete project directory: {}",
+                folder.to_str().unwrap()
+            ));
+        }
+        folder
+    };
+
+    // $ cargo run -- new demoproject_12 -d pont
+    // $ dove new demoproject_12 -d pont
+    {
+        let args = &["dove", "new", project_name];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, dove_folder.clone()).expect(&format!("[COMMAND] {}", &command_string));
+        set_dependencies_local_move_stdlib(&project_folder);
+    }
+
+    // $ cargo run -- build
+    // $ dove build
+    {
+        let args = &["dove", "build"];
+        let command_string: String = args.join(" ").to_string();
+        execute(args, project_folder.clone()).expect(&format!(
+            "[COMMAND] {}\r\n[FOLDER] {}",
+            &command_string,
+            project_folder.to_str().unwrap()
+        ));
+    }
+
+    // project_folder/tests/test_1.move
+    {
+        let test_1_path = {
+            let mut path = project_folder.clone();
+            path.push("tests");
+            path.push("test_1.move");
+            path
+        };
+        write_all(
+            &test_1_path,
+            "script {
+                    fun main() {
+                        assert((3+2)==4,1);
+                    }
+                }",
+        )
+        .unwrap();
+    }
+    // $ cargo run -- test
+    // $ dove test
+    {
+        let args = &["dove", "test"];
+        let command_string: String = args.join(" ").to_string();
+        assert!(
+            execute(args, project_folder.clone()).is_err(),
```

Comment:
nit: unlikely, but test `dove test` can fail for another reason, we should probably check error message